pkg/volume/csi/csi_attacher_test.go

```diff
@@ -1559,6 +1559,7 @@ func newTestWatchPlugin(t *testing.T, fakeClient *fakeclient.Clientset) (*csiPlu
 		ProbeVolumePlugins(),
 		"fakeNode",
 		csiDriverLister,
+		nil,
```

Comment:
Can you open another issue to track updating these unit tests to be able to work with informers?

Reply:
Done. [Issue#91784](https://github.com/kubernetes/kubernetes/issues/91784)